earth_enterprise/src/fusion/autoingest/sysman/khResourceManager.cpp

```diff
@@ -416,6 +423,13 @@ khResourceManager::TryActivate(void) throw()
       }
     }
 
+    for (const auto& p : providers)
```

Comment:
Is this meant to replace the loop above? If so, we need to remove the original version.